weave/integrations/langchain/cassettes/langchain_test/test_simple_rag_chain.yaml

```diff
@@ -0,0 +1,11531 @@
+interactions:
+- request:
```

Comment:
This first chunk looks fishy to me, probably remove

Reply:
removed